src/model/formats/format-utils.ts

```diff
@@ -38,10 +38,8 @@ export class FormatUtils {
 
   public static promptToPartialChecklistItem(prompt: string): Partial<ChecklistItem> {
     const [prefix, rest] = FormatUtils._splitByColon(prompt);
-    const itemType = FormatUtils.CHECKLIST_ITEM_TYPES.get(prefix);
-    return itemType !== undefined
-      ? { type: itemType, prompt: rest }
-      : { type: ChecklistItem_Type.ITEM_PLAINTEXT, prompt: prompt };
+    const itemType = FormatUtils.CHECKLIST_ITEM_TYPES.get(prefix) ?? ChecklistItem_Type.ITEM_PLAINTEXT;
+    return rest !== '' ? { type: itemType, prompt: rest } : { type: ChecklistItem_Type.ITEM_SPACE };
```

Comment:
Hmmm this feels wrong - if I create a text item with the contents "Foo:bar", that'll become a plaintext item with just bar. The previous code used the whole `prompt` (not just `rest`) for that reason.

Reply:
Removed, see last comment in #195.